components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -142,25 +143,26 @@ def _generate_clp_io_config(
             path_prefix_to_remove=str(CONTAINER_INPUT_LOGS_ROOT_DIR),
         )
     elif InputType.S3 == input_type:
-        if len(logs_to_compress) != 1:
-            raise ValueError(f"Too many key prefixes: {len(logs_to_compress)} > 1")
-
-        s3_config = clp_config.logs_input.s3_config
+        if len(logs_to_compress) == 0:
```

Comment:
Is this case possible for InputType.FS as well?

Reply:
technically yes, added check for FS flow